catt/util.py

```diff
@@ -38,6 +38,9 @@ def echo_status(status):
     if status.get("volume_level"):
         click.echo("Volume: {}".format(status["volume_level"]))
 
+    if status.get("volume_muted"):
+        click.echo("Volume Muted: {}".format(status["is_volume_muted"]))
```

Comment:
```suggestion
        click.echo("Volume muted: {}".format(status["is_volume_muted"]))
```

Reply:
I chose `volumemute` because the attribute from pychromecast data is `volume_muted`. It just seemed more consistent. I'm not particularly attached to it, so changing it to `mute` would also be fine.